cmd/earthly/satellite_cmds.go

```diff
@@ -430,6 +567,24 @@ func (app *earthlyApp) actionSatelliteRemove(cliCtx *cli.Context) error {
 		return err
 	}
 
+	satellites, err := cloudClient.ListSatellites(cliCtx.Context, orgID, true)
+	if err != nil {
+		return err
+	}
+
+	found := false
+	for _, s := range satellites {
+		if app.satelliteName == s.Name {
+			found = true
+			if s.Hidden {
+				return errors.New("cannot delete hidden satellites")
+			}
+		}
+	}
+	if !found {
+		return fmt.Errorf("could not find '%s' for deletion", app.satelliteName)
```

Comment:
I just noticed that the other bits of this code already uses single quotes elsewhere... I'll leave it up to you in case you want to switch all of them (or none)

Reply:
Should it be `errors.Wrap` since earthly-core usually opts for `pkg/errors` ?

Reply:
I think that ship sailed -- we have both `errors.Errorf` and `fmt.Errorf` all over the place now. Note that this specific case isn't wrapping an error -- so it's less important.

Reply:
For some reason my brain doesn't remember that `%q` is a thing. Ill fix those.